### ISSUE TITLE
Style for input range doesn't work in asp.net

### ISSUE THREAD

**Issue by ns8482e** [beercss/beercss#400]
Using slider in asp.net styles are not rendered as expected, asp.net input tag helper generates an additional hidden input `__Invariant`

Using following cshtml 
```cshtml
<label asp-for="Value" >
    <input asp-for="Value" type="range" placeholder=" " />
    <span></span>
    <div class="tooltip"></div>
</label>
```
Generate following view

<img width="115" alt="image" src="https://github.com/user-attachments/assets/a0a1502e-152f-4018-8d69-41027dab87e2" />

Workaround is to add custom css as below

```css
  .slider>input+input[type=hidden]~span {
      border-radius: 1rem 0 0 1rem;
      clip-path: polygon(0 0, calc(100% - .5rem) 0, calc(100% - .5rem) 100%, 0 100%);
    }
```

Example here https://codepen.io/ns8482e/pen/dPbOXvp

**Comment by leonardorafael** [comment]
Nice, I think we could add this on next release. 👍 

I don't know why asp.net adds a hidden input element, when you can get the value directly from the range input element. Maybe not, better not know.

It will be available on next release, thanks!

**Comment by leonardorafael** [comment]
Available in [v3.8.0](https://github.com/beercss/beercss/releases/tag/v3.8.0)
